### ISSUE TITLE
Issues trying to download One Piece English subbed

### ISSUE THREAD

**Issue by hungLink** [NobilityDeviant/ZenDownloader#8]
I queue every episode, but it only downloads about 8-12 episodes before gracefully finishing.

episode 505 won't download:
[10:14] Failed to find video frame for https://wcofun.net/one-piece-episode-505-english-subbed. Retrying...

Some episodes fail to determine size and don't download:
[10:01] Retrying... Failed to determine file size for: One Piece episode 501 English Subbed

**Comment by hungLink** [comment]
Sometimes it gets stuck trying to find the video frame for an episode that is already stored locally.
If (this ticket)[https://github.com/NobilityDeviant/ZenDownloader/issues/9] was implemented, it might bypass this part of this issue.

**Comment by NobilityDeviant** [comment]
Yes i'm working on the file size issue, but it's hard to reproduce for me. I do have some ideas which would address both issues potentially.

**Comment by NobilityDeviant** [comment]
I will officially address this next update. I'm very sorry for ignoring it, but I honestly couldn't figure it out.
Also the one piece title updating is going to have to not be implemented. That will take too much time and i'd rather leave it to the individual. I will try to implement a way to mass update episode names,

**Comment by hungLink** [comment]
It's okay! I discovered and started watching one pace. It's overall better than kaizoku, even if it's too fast at times.


Sent with Proton Mail secure email.
